backend/plugins/cars_api/src/modules/cars/@types/category.ts

```
@@ -0,0 +1,18 @@
+import { Document } from 'mongoose';
+import { IAttachment } from 'erxes-api-shared/core-types';
+
+export interface ICarCategory {
+  name: String;
```

Biome (1.9.4)

[error] Don't use 'String' as a type.

Use lowercase primitives for consistency.
Safe fix: Use 'string' instead


(lint/complexity/noBannedTypes)